docs/notebooks/simulation.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulation with JuPedSim\n",
+    "\n",
+    "In this tutorial a simulation with `jpscore` is performed in a simple setup. \n",
+    "\n",
+    "A documentation of the used files is under:\n",
+    "\n",
+    "- The geometry file (see [documentation](http://www.jupedsim.org/jpscore/jpscore_geometry.html)) \n",
+    "- The project file of jpscore, called inifile (see [documentation](http://www.jupedsim.org/jpscore/jpscore_inifile.html))\n",
+    "\n",
+    "### Before starting\n",
+    "\n",
+    "- Open file `config.ipynb`\n",
+    "- Edit the values of `jpscore_inifile` and `jpscore_dir`\n",
+    "- Save\n",
+    "\n",
+    "### How to run\n",
+    "\n",
+    "`Run --> Run All Cells` \n",
+    "\n",
+    "-----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load config files\n",
+    "\n",
+    "the used inifiles as defined by the **user** in config.ipynb are loaded"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "%run config.ipynb\n",
+    "%store -r jpscore_exe\n",
+    "%store -r jpscore_inifile\n",
+    "%store -r jpscore_ini_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start the simulation\n",
+    "\n",
+    "1. start simulation with jpscore\n",
+    "2. move *results* to the direcory of the inifile"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
This cell is something for the advanced user.

Reply:
Small errors can block the execution.

Reply:
What errors? Did you encounter any?

Reply:
do you mean this cell?

<img width="636" alt="Screenshot 2020-05-12 02 54 16" src="https://user-images.githubusercontent.com/5772973/81626414-ea961480-93fb-11ea-8af4-2ea975549551.png">

Hmm, it looks a bit cryptic, but the user is not supposed to touch it. In fact, the whole notebook is meant to be run at one `Run All Cells`.

Reply:
Do you know your users? If you don't want the user to touch these lines, why don't you  move them into a function?

Reply:
I don't find an easy way to do so. A script can be an idea, but then I have to make it executable and this is something not good. 

I trust that the users read the very first cell and does not mess up.  :-)

Reply:
It's ok. Thanks.